### ISSUE TITLE
count option use causes inconsistence 

### ISSUE THREAD

**Issue by masciugo** [svenfuchs/i18n#303]
Hi, is this consistent?
    
   irb(main):001:0> require 'I18n'
    true
    irb(main):002:0> I18n.backend.store_translations(:en,{cube:nil,ball:{one:nil,other:nil}})
    {
        :cube => nil,
        :ball => {
              :one => nil,
            :other => nil
        }
    }
    irb(main):003:0> I18n.default_locale = :en
    :en
    irb(main):004:0> I18n.t(:cube)
    "translation missing: en.cube"
    irb(main):005:0> I18n.t(:ball, count: 1)
    nil

Should not both translations return the same?

**Comment by radar** [comment]
Yes I believe they should both return the same. I'm open to a PR to fix this issue.
